doc/conf.py

```diff
@@ -14,22 +14,23 @@
 
 import os
 import sys
-sys.path.insert(0, os.path.abspath('.'))
+
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
I'm OK with being consistent and using `"` for the strings, but just want to check that the linter still permits us to use `'` when the string itself includes a double quote?

I.e., do we follow https://www.python.org/dev/peps/pep-0008/#string-quotes?

Reply:
Yeah, Black will ignore that rule if there are double quotes within a string.